datahub-web-react/src/app/entity/dashboard/DashboardEntity.tsx

```diff
@@ -153,9 +154,11 @@ export class DashboardEntity implements Entity<Dashboard> {
         // TODO: Get rid of this once we have correctly formed platform coming back.
         const name = dashboard?.properties?.name;
         const externalUrl = dashboard?.properties?.externalUrl;
+        const subTypes = dashboard?.subTypes;
         return {
             name,
             externalUrl,
+            entityTypeOverride: subTypes ? capitalizeFirstLetterOnly(subTypes.typeNames?.[0]) : '',
```

Comment:
Minor: We may want to check that subTypes.length > 0.

If we don't do this elsewhere, we should!

Reply:
Actually this would be fine. Since capitalizeFirstLetterOnly can handle a missing value.